com.brave.Browser.metainfo.xml

```diff
@@ -5,10 +5,14 @@
   <launchable type="desktop-id">com.brave.Browser.desktop</launchable>
   <name>Brave</name>
   <developer_name>Brave Software</developer_name>
-  <summary>Fast Internet, AI, Adblock</summary>
+  <summary>Block ads. Save data. And get way faster webpages.</summary>
```

Comment:
This can be reverted. Should be under 35 characters but ideally 10 to 25 as per https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/quality-guidelines/#summary

Reply:
done